### PR TITLE
Addresses BACKLOG-40066: New schedule execution permission

### PR DESCRIPTION
**Pull request by mbrasil**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 1m 25s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,ui
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,ui
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: core/src/main/java/org/pentaho/platform/web/http/api/resources/SchedulerResource.java [hunk 1]
```diff
@@ -607,6 +607,32 @@ public String doGetCanSchedule() {
     return schedulerService.doGetCanSchedule();
   }
 
+  /**
+   * Checks whether the current user has authority to execute a schedule in the platform.
```
Comment (afrjorge) [inline core/src/main/java/org/pentaho/platform/web/http/api/resources/SchedulerResource.java:611]: ~a schedule~ all the schedules

File: ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPerspectivePanel.java [hunk 2]
```diff
@@ -56,39 +58,65 @@ public SchedulesPerspectivePanel() {
...
+      requestBuilder.setHeader( IF_MODIFIED_SINCE, IF_MODIFIED_SINCE_DATE );
+      requestBuilder.sendRequest( null, new RequestCallback() {
+        public void onError( Request request, Throwable caught ) {
+          isScheduler = false;
+          canExecuteSchedules();
+        }
+
+        public void onResponseReceived( Request request, Response response ) {
+          isScheduler = "true".equalsIgnoreCase( response.getText() ); //$NON-NLS-1$
+          canExecuteSchedules();
```
Comment (afrjorge) [inline ui/src/main/java/org/pentaho/mantle/client/workspace/SchedulesPerspectivePanel.java:96]: it is redundant to check if the user can execute schedules when he already has the highest permission for managing schedules, which includes executing schedules

**Comment by buildguy** [comment]
### :x: Build failed in 2m 48s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,ui
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,ui
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -Djs.no.sandbox -pl core,ui
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by mbrasil** [comment]
This PR is wrong and contains code from other stories. I will close this and open others with the same code.

**Pull request closed without merging**